src/solver/svd/svd.cpp

```diff
@@ -3,7 +3,7 @@
 
 namespace neon
 {
-void bdc_svd(void) {} // I don't know why this is here
+void bdc_svd(void) {} // I don't know why this is here (keep or delete)
```

Comment:
You don't need to specify `void` here, or the constructor if it's empty.  See below if you want a default constructor.